frontend/packages/ceph-storage-plugin/src/components/ocs-install/internal-mode/install-wizard.tsx

```diff
@@ -84,41 +83,23 @@ export const CreateInternalCluster: React.FC<CreateInternalClusterProps> = ({
   const hasEnabledCreateStep =
     !!(state.nodes.length >= MINIMUM_NODES && scName && state.kms.hasHandled) &&
     hasConfiguredNetwork;
-  const { getStep, getParamString, getIndex, getAnchor } = navUtils;
-
-  const toLink = (steps: any, currentStep: string, modes: any) =>
-    getAnchor(getIndex(steps, currentStep, 2), getIndex(modes, modes.INTERNAL));
+  const { getStep, getParamString, getIndex } = navUtils;
 
   const steps: WizardStep[] = [
     {
-      name: (
-        <Link to={toLink(CreateStepsSC, CreateStepsSC.STORAGEANDNODES, MODES)}>
-          {' '}
-          {t('ceph-storage-plugin~Capacity and nodes')}{' '}
-        </Link>
-      ),
+      name: t('ceph-storage-plugin~Select capacity and nodes'),
```

Comment:
It is `Capacity and nodes` and IIRC `Link` was added because it gave navItems clickable link like UI (otherwise these items will just display as black text). So, if we are removing this we should confirm with UX once ??

Reply:
IIRC `Link` also provided clickable link like UI to these navItems (items were displayed blue in color, otherwise they will be displayed as a black text).